apps/browser/qml/pages/components/BrowserListItem.qml

```diff
@@ -0,0 +1,57 @@
+import QtQuick 2.0
+import Sailfish.Silica 1.0
+
+ListItem {
+    id: root
+
+    property alias label: valueButton.label
+    property alias value: valueButton.value
+    property alias iconSource: icon.source
+    property alias secondaryText: secondaryLabel.text
+
+    width: parent.width
+    contentHeight: Math.max(Theme.itemSizeMedium, row.height)
+    onClicked: openMenu()
+
+    Row {
+        id: row
+
+        width: parent.width - 2*Theme.horizontalPageMargin
+        x: Theme.horizontalPageMargin
+        anchors.verticalCenter: parent.verticalCenter
+
+        Icon {
+            id: icon
+
+            anchors.verticalCenter: parent.verticalCenter
+        }
+
+        Column {
+            width: parent.width
+            anchors.verticalCenter: parent.verticalCenter
+
+            ValueButton {
```

Comment:
`ValueButton` will apply an indent of `Theme.horizontalPageMargin` by default, so this should be set to `Theme.paddingMedium` in this case to align with `secondaryLabel`:
```
leftMargin: Theme.paddingMedium
```

Reply:
I'm unresolving this as I still see the additional space here I'm afraid. According to the design, the "Save destination" text should align with the text "Permissions" directly above, but currently the "Save destination" text is too far to the right.